v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of multiple independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single transaction. Parallel execution of SQL statements reduces the overall execution time of multiple SQL statements.
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+To maintain strong consistency across a globally-distributed cluster, CockroachDB replicates writes using the [consensus protocol](https://www.cockroachlabs.com/blog/consensus-made-thrive/). The protocol requires majority of replicas to agree on a write before committing it. Thus writes to CockroachDB require coordination between globally-distributed nodes. Communication between these nodes takes time and contributes to communication latency. Executing SQL statements sequentially, as in traditional SQL engines, leads to higher cumulative communication latency.
+
+Parallel execution of statements can help reduce the communication latency. With parallel statement execution, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency over multiple SQL statements. 
```

Comment:
This paragraph is nice but feels a little bit redundant in places. I'd suggest trimming it down to:

> With parallel statement execution, however, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency.